src/vm/excep.cpp

```diff
@@ -5152,10 +5138,20 @@ LONG InternalUnhandledExceptionFilter(
 
 } // LONG InternalUnhandledExceptionFilter()
 
+static bool useEntryPointFilter = false;
```

Comment:
Nit: We typically prefix simple statics with `s_` prefix. (The codebase is not consistent though.)